tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -194,12 +207,29 @@ declare var cfg:ISnippetConfig;
                 });
                 // let message = "Load Version 2 SDK instead to support IE"; // where to report this error?
             }
+
+            if (cfg.cr){
```

Comment:
I think we want this to be `if (cfg.cr !== false) {` so that when anything else (undefined, true, etc) it will perform the retry